recipes-sota/sota-client-cpp/sota-client-cpp_git.bb

```diff
@@ -0,0 +1,29 @@
+SUMMARY = "SOTA Client"
+DESCRIPTION = "SOTA Client application written in C++"
+HOMEPAGE = "https://github.com/advancedtelematic/sota_client_cpp"
+SECTION = "base"
+
+LICENSE = "MPL-2.0"
+LIC_FILES_CHKSUM = "file://${S}/LICENSE;md5=9741c346eef56131163e13b9db1241b3"
+
+inherit cmake systemd
+
+S = "${WORKDIR}/git"
+
+SRCREV = "7194ec4aa39f71157af66751098340c854537817"
+
+SRC_URI = " \
+	git://github.com/advancedtelematic/sota_client_cpp \
+	"
+
+DEPENDS = "boost curl openssh"
```

Comment:
Yes, thank you for pointing out this bug. I will fix it right away.

Best regards,
Leon